src/gh/diffCheck/diffCheck/df_error_estimation.py

```diff
@@ -0,0 +1,156 @@
+#! python3
+"""
+    This module contains the utility functions to compute the difference between source and target
+"""
+
+import numpy as np
+from diffCheck import diffcheck_bindings
+import Rhino.Geometry as rg
+
+
+def cloud_2_cloud_comparison(source_list, target_list):
```

Comment:
maybe for all the naming of this modules we could use the convention of specifying if it is `df` or `rh` since we are doing it for the mesh but not the cloud. Later we might take directly df_clouds and have overloads for those. So here would be: `rh_cloud_2_rh_cloud_comparison` (?)

Reply:
switched to:
df_cloud_2_df_cloud_comparison()
df_cloud_2_rh_mesh_comparison()
rh_mesh_2_df_cloud_distance()
df_cloud_2_rh_mesh_distance()